torch/jit/_script.py

```diff
@@ -73,9 +73,10 @@ def Attribute(value, type):  # type: ignore[no-redef]
 
     Though TorchScript can infer correct type for most Python expressions, there are some cases where
     type inference can be wrong, including:
-    - Empty containers like `[]` and `{}`, which TorchScript assumes to be container of `Tensor`s
+
+    - Empty containers like `[]` and `{}`, which TorchScript assumes to be container of `Tensor`\ s
```

Comment:
Same as comment to torch/jit/__init__.py
```suggestion
    - Empty containers like `[]` and `{}`, which TorchScript assumes to be container of *Tensors*
```